### PR TITLE
MultiSelect/ FilterBar Feature Branch

### PR DESCRIPTION
**Pull request by isastettler**
Fixes JIRA tickets [DSD-240](https://jira.nypl.org/browse/DSD-240), [DSD-957](https://jira.nypl.org/browse/DSD-957), [DSD-234](https://jira.nypl.org/browse/DSD-234).

## This PR does the following:

-

## Accessibility concerns or updates

-

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Let's review this one more time. Isa pointed out that the updated `Button` styles (for the sizes) altered the `MultiSelect`.

File: src/theme/components/multiSelectMenuButton.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const MultiSelectMenuButton = {
```
Comment (bigfishdesign13) [inline src/theme/components/multiSelectMenuButton.ts:3]: Add the following to the top of the doc…
```
import { defaultElementSizes } from "./global”;
```

File: src/theme/components/multiSelectMenuButton.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      transition: "margin 150ms ease-out",
+    },
+    menuButton: {
+      alignItems: "center",
+      backgroundColor: isOpen ? "ui.gray.x-light-cool" : "ui.white",
+      borderBottomLeftRadius: isOpen ? "0" : "button.default",
+      borderBottomRightRadius: isOpen ? "0" : "button.default",
+      borderColor: isOpen ? "ui.gray.dark" : "ui.gray.medium",
+      borderWidth: "1px",
+      padding: "8px 8px 7px 16px",
```
Comment (bigfishdesign13) [inline src/theme/components/multiSelectMenuButton.ts]: Change `7px` to `8px`.
```
padding: "8px 8px 8px 16px”,
```

File: src/theme/components/multiSelectMenuButton.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      justifyContent: "flex-start",
+      overflow: "hidden",
+      marginLeft: hasSelectedItems ? "50px" : "0",
+      marginRight: "l",
+      textAlign: "left",
+      textOverflow: "ellipsis",
+      whiteSpace: "nowrap",
+      transition: "margin 150ms ease-out",
+    },
+    menuButton: {
```
Comment (bigfishdesign13) [inline src/theme/components/multiSelectMenuButton.ts:21]: Add the following attributes to this style block...
```
borderRadius: "button.default",
fontSize: "button.default",
minHeight: { base: defaultElementSizes.mobileFieldHeight, md: "auto" },
```

File: src/theme/components/multiSelectMenuButton.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      borderRadius: "20px",
+      borderColor: "ui.gray.medium",
+      display: "flex",
+      flexShrink: 0,
+      fontSize: "text.tag",
+      justifyContent: "flex-end",
+      left: "15px",
+      marginRight: "xs",
+      position: "absolute",
+      top: "10px",
```
Comment (bigfishdesign13) [inline src/theme/components/multiSelectMenuButton.ts]: Update to the following…
```
top: { base: "12px", md: "10px" },
```

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Some CSS adjustments are needed.  Please see my comments.

**Review by bigfishdesign13 — APPROVED** [review]
Updates look good.

**Review by bigfishdesign13 — APPROVED** [review]
The feature branch looks good to me.

File: src/hooks/__tests__/useFilterBar.test.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import { renderHook, act } from "@testing-library/react-hooks";
+import useFilterBar from "../useFilterBar";
+
+describe("usFilterBar hook", () => {
```
Comment (EdwinGuzman) [inline src/hooks/__tests__/useFilterBar.test.ts]: Minor typo.

**Review by EdwinGuzman — APPROVED** [review]
Looks good. Approving but it needs a minor update to the CHANGELOG to include the three new additions.

File: src/hooks/__tests__/useFilterBar.test.ts (hunk 5 above)
Comment (isastettler) [reply src/hooks/__tests__/useFilterBar.test.ts]: fixed

**Pull request merged**